rust_src/remacs-sys/lib.rs

```diff
@@ -522,6 +522,8 @@ pub struct Lisp_Buffer {
     pub overlay_center: ptrdiff_t,
 
     pub undo_list: Lisp_Object,
+
+    pub prevent_redisplay_optimizations_p: bool,
```

Comment:
This is not the right location for this member - it is part of the `flags` bitfield declared above.

Since Rust doesn't support bitfields in structs, you'll have to extract it from there.

Reply:
ok i'll look into fixing this